lbrynet/daemon/Downloader.py

```diff
@@ -89,22 +88,21 @@ def check_status(self):
         else:
             log.debug("Waiting for stream descriptor (%i seconds)", self.timeout_counter)
 
-    def convert_max_fee(self):
-        currency, amount = self.max_key_fee['currency'], self.max_key_fee['amount']
-        return self.exchange_rate_manager.convert_currency(currency, "LBC", amount)
+    def convert_max_fee(self, fee):
+        return self.exchange_rate_manager.convert_currency(fee.currency, "LBC", fee.amount)
 
     def set_status(self, status, name):
         log.info("Download lbry://%s status changed to %s" % (name, status))
         self.code = next(s for s in STREAM_STAGES if s[0] == status)
 
     @defer.inlineCallbacks
     def check_fee_and_convert(self, fee):
-        max_key_fee_amount = self.convert_max_fee()
-        converted_fee_amount = self.exchange_rate_manager.convert_currency(fee.currency, "LBC",
-                                                                           fee.amount)
+        max_key_fee_amount = self.convert_max_fee(self.max_key_fee)
+        converted_fee_amount = self.convert_max_fee(fee.currency, fee.amount)
```

Comment:
Can't see how this could work. Did you run this code?

Reply:
Fixed it. Wrong version.